libraries/dotnet/EnergyOrigin.TokenValidation/EnergyOrigin.TokenValidation/B2C/IServiceCollectionExtensions.cs

```diff
@@ -53,6 +53,15 @@ public static void AddB2CAndTokenValidation(this IServiceCollection services, B2
                 .Build();
             options.AddPolicy(Policy.B2CPolicy, b2CPolicy);
 
+            var b2CTosAcceptedPolicy = new AuthorizationPolicyBuilder()
+                .RequireAuthenticatedUser()
+                .AddAuthenticationSchemes(
+                    AuthenticationScheme.B2CClientCredentialsCustomPolicyAuthenticationScheme,
+                    AuthenticationScheme.B2CMitIDCustomPolicyAuthenticationScheme)
+                .RequireClaim(ClaimType.TosAccepted, "true")
```

Comment:
This should just be part of already existing policies. This should be part of SubTypeUser.

Reply:
Maybe this shouldn't be alone, since it will break everything until we have terms up and running.

Reply:
Making it a part of an existing policy, would result in a breaking change. this way, consumers of the nuget package, can opt-in, for the added functionality. 

I would rather have the functionality available like this, and when the feature is finished being implemented, we can talk about including it in the current policy.

Reply:
Martin, You and me just had a talk about this issue? We agreed upon extending existing policy. No need to add a policy just to delete it afterwards again? The frontend is not using the new endpoint anyway, so not like we are blocking anyone. This will just add legacy code out of the gate.

Reply:
Maybe this shouldn't be alone, since it will break everything until we have terms up and running. <-- This PR is not self contained. We can't use the policy as is. We need endpoints, logic, etc. We had this talk about the big issue being a 13 pointer and would take weeks.

Reply:
There is no breaking change, as long as the package is not updated in the subsystems

Reply:
Exactly, so we don't have the option of making other changes in this package before the rest of the functionality has been delivered. I agree that this part could be done last, when all functionality regarding the new claim and accepting terms have been delivered.